cmd/driver/config/config.go

```diff
@@ -108,12 +128,6 @@ func checkFalcoRunsWithDrivers(engineKind string) error {
 
 func (o *driverConfigOptions) replaceDriverTypeInFalcoConfig(driverType drivertype.DriverType) error {
 	falcoCfgFile := filepath.Clean(filepath.Join(string(os.PathSeparator), "etc", "falco", "falco.yaml"))
-	type engineCfg struct {
```

Comment:
Now used from both `replaceDriverTypeInFalcoConfig` and `replaceDriverTypeInK8SConfigMap`.